### PR TITLE
add osx_arm64 support

### PR DESCRIPTION
**Pull request by matthiasdiener**
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [ ] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by looooo** [comment]
@matthiasdiener Serena like there is a dependency conflict. Any ideas. Is vtk already built for osx_arm64?

**Comment by matthiasdiener** [comment]
> @matthiasdiener Serena like there is a dependency conflict. Any ideas. Is vtk already built for osx_arm64?

An osx_arm64 package is still missing for vtk (and some of its dependencies), I'm working on those.

File: recipe/meta.yaml [hunk 1]
```diff
@@ -37,6 +37,11 @@ outputs:
         - {{ cdt('libxxf86vm') }}        # [linux]
         - {{ cdt('mesa-libgl-devel') }}  # [linux]
         - {{ cdt('mesa-dri-drivers') }}  # [linux]
+        - python                                 # [build_platform != target_platform]
+        - cross-python_{{ target_platform }}     # [build_platform != target_platform]
+        - cython                                 # [build_platform != target_platform]
+        - numpy                                  # [build_platform != target_platform]
+        - pybind11                               # [build_platform != target_platform]
```
Comment (isuruf) [inline recipe/meta.yaml:44]:
```suggestion
```

File: recipe/meta.yaml [hunk 2]
```diff
@@ -37,6 +37,11 @@ outputs:
         - {{ cdt('libxxf86vm') }}        # [linux]
         - {{ cdt('mesa-libgl-devel') }}  # [linux]
         - {{ cdt('mesa-dri-drivers') }}  # [linux]
+        - python                                 # [build_platform != target_platform]
```
Comment (isuruf) [inline recipe/meta.yaml:40]:
```suggestion
        - python *
```

**Pull request merged**